pkg/build/registry/buildconfig/strategy.go

```diff
@@ -80,7 +80,9 @@ func SelectableFields(buildConfig *api.BuildConfig) fields.Set {
 
 // CheckGracefulDelete allows a build config to be gracefully deleted.
 func (strategy) CheckGracefulDelete(obj runtime.Object, options *kapi.DeleteOptions) bool {
-	return false
+	options.GracePeriodSeconds = new(int64)
+	*options.GracePeriodSeconds = 0
+	return true
```

Comment:
what is this actually accomplishing in terms of the buildconfig delete flow/behavior?